chia/util/db_wrapper.py

```diff
@@ -7,6 +7,11 @@
 import aiosqlite
 
 
+if aiosqlite.sqlite_version_info < (3, 32, 0):
```

Comment:
I think ensuring that this comparison is correct is key to the correctness of this change

Reply:
That's what I tested:

```python
>>> import aiosqlite
>>> aiosqlite.sqlite_version_info
(3, 34, 1)
>>> if aiosqlite.sqlite_version_info < (3, 32, 0):
...   SQLITE_MAX_VARIABLE_NUMBER = 999
... else:
...   SQLITE_MAX_VARIABLE_NUMBER = 32766
...
>>> print(SQLITE_MAX_VARIABLE_NUMBER)
32766
>>> if aiosqlite.sqlite_version_info < (3, 4, 0):
...   SQLITE_MAX_VARIABLE_NUMBER = 999
... else:
...   SQLITE_MAX_VARIABLE_NUMBER = 32766
...
>>> print(SQLITE_MAX_VARIABLE_NUMBER)
32766
>>> if aiosqlite.sqlite_version_info > (3, 32, 0):
...   SQLITE_MAX_VARIABLE_NUMBER = 999
... else:
...   SQLITE_MAX_VARIABLE_NUMBER = 32766
...
>>> print(SQLITE_MAX_VARIABLE_NUMBER)
999
```